crates/fluvio-cluster/src/start/k8.rs

```diff
@@ -579,8 +580,6 @@ pub struct ClusterInstaller {
     config: ClusterConfig,
     /// Shared Kubernetes client for install
     kube_client: SharedK8Client,
-    /// Helm client for performing installs
-    helm_client: HelmClient,
```

Comment:
This is not being used,  do we have to use this?

Reply:
let's remove unused